packages/e2e-tests/src/specs/dashboard/dashboard.js

```diff
@@ -45,7 +45,8 @@ describe('Stories Dashboard', () => {
     await takeSnapshot(page, 'Stories Dashboard', { percyCSS });
   });
 
-  it('should be able to skip to main content of Dashboard for keyboard navigation', async () => {
+  //eslint-disable-next-line jest/no-disabled-tests -- https://github.com/GoogleForCreators/web-stories-wp/runs/7301628196?check_suite_focus=true#step:9:19
```

Comment:
Let's not link to individual runs. These logs disappear after a while.

Let's write a short explanation instead + create a ticket to fix these skipped tests.